docs/content/filen.md

```diff
@@ -0,0 +1,265 @@
+---
+title: "Filen"
+description: "Rclone docs for Filen"
+versionIntroduced: "TODO"
+status: Alpha
```

Comment:
I'd leave out the status. If it passes all the integration tests then it is much better than Alpha :-)